Cargo.toml

```diff
@@ -4,3 +4,11 @@ members = [
 	"selene",
     "selene-lib",
 ]
+
+[workspace.package]
+version = "0.21.1"
+edition = "2021"
+homepage = "https://kampfkarren.github.io/selene/"
+license = "MPL-2.0"
+repository = "https://github.com/Kampfkarren/selene"
+rust-version = "1.64"

```

Comment:
Don't include this please, I never want to commit to an MSRV.

Reply:
selene is basically never guaranteed to compile on anything other than latest because I always like using new features that come out, and so I'd always be needing to update this, making it fairly useless for anyone trying to actually rely on it.

Reply:
Fair enough, removed!